Userland/Utilities/beep.cpp

```diff
@@ -7,13 +7,20 @@
 #include <LibCore/ArgsParser.h>
 #include <LibCore/System.h>
 #include <LibMain/Main.h>
+#include <stdint.h>
 
 ErrorOr<int> serenity_main(Main::Arguments arguments)
 {
-    Optional<size_t> tone;
+    size_t tone = 0;
     Core::ArgsParser args_parser;
-    args_parser.add_option(tone, "Beep tone", "beep-tone", 'f', "Beep tone (frequency in Hz)");
+    args_parser.add_positional_argument(tone, "Beep tone (frequency in Hz)", "beep-tone", Core::ArgsParser::Required::No);
     args_parser.parse(arguments);
-    TRY(Core::System::beep(tone));
+    if (tone == 0) {
+        TRY(Core::System::beep());
+    } else {
+        if (tone > UINT16_MAX)
+            return Error::from_string_literal("Invalid tone - higher than 65535");
+        TRY(Core::System::beep(tone));
+    }
```

Comment:
We should probably just have the variable initialized to a sensible default (probably 440, just like the `beep` function?) and then skip the `tone == 0` check.

```suggestion
    size_t tone = 440;
    
    // [ArgsParser will leave the default alone if the option isn't passed.]
    // [...]

    if (tone > UINT16_MAX)
        return Error::from_string_literal("Invalid tone - higher than 65535");
    TRY(Core::System::beep(tone));
```